tests/helpers/module-for-acceptance.js

```diff
@@ -3,8 +3,7 @@ import { resolve } from 'rsvp';
 import startApp from '../helpers/start-app';
```

Comment:
Can we not move to the new qunit style? Separate PR maybe?

Reply:
This is a required change applied automatically by the blueprint to support the major version bumps to qunit-related dependencies, afaik.

Reply:
Ya sure. I guess my point was that we should update this addon fully while we're doing all this. I'm fine if it's in a separate PR.

Reply:
I guess I just don't see the value in breaking apart a set of changes we get for free from the blueprint itself.

Reply:
Do we get updated test APIs for free?

Reply:
Chatted offline, seems like something we can run as a separate PR but definitely should get done.